R/labeller.R

```diff
@@ -0,0 +1,26 @@
+#' A labeller function to parse TeX syntax
+#'
+#' Thi function formats the strip labels of facet grids and wraps that contains
+#' TeX expressions.
+#'
+#' @seealso [ggplot2::labeller], [latex2exp::TeX()]
+#'
+#' @inheritParams ggplot2::label_parsed
+#' @inheritDotParams ggplot2::label_parsed -labels
+#'
+#' @examples
+#' library(ggplot2)
+#' d <- data.frame(x = 1, y = 1, facet = "$\\beta$")
+#' ggplot(d, aes(x, y)) +
+#'  geom_point() +
+#'  facet_wrap(~ facet, labeller = label_tex)
+#'
+#' @importFrom latex2exp TeX
```

Comment:
This needs to be removed if in Suggest

Reply:
maybe we can wrap into unreachable if statement, so that we can still show the example code?

```
if (FALSE) {
  ...
}
```

Reply:
it's the `@importFrom` - not the example

Reply:
it's the `@importFrom` - not the example

Reply:
oh, but I still have to remove here.

> @importFrom latex2exp TeX

Reply:
yes - remove `@importFrom ...` - keep the example